src/providers/spatialite/qgsspatialiteprovider.cpp

```diff
@@ -44,6 +44,8 @@ email                : a.furieri@lqt.it
 #include <QDir>
 #include <QRegularExpression>
 
+#include <stdexcept>
```

Comment:
not needed anymore?

Reply:
Depends on the resolution of the other comment, but I have followed the advice there in the current version so it's gone at the moment.